src/main/kotlin/cc/hyperium/utils/ResourceManager.kt

```diff
@@ -0,0 +1,9 @@
+package cc.hyperium.utils
+
+import net.minecraft.util.ResourceLocation
+
+class ResourceManager : HashMap<String, ResourceLocation>() {
+    fun get(k: String, def: ResourceLocation?) : ResourceLocation {
+        return computeIfAbsent(k){ def!! }
```

Comment:
What is the point of this function? Why would the user not just call `computeIfAbsent` directly if thats what the situation calls for?